### ISSUE TITLE
Still Connecting To Bungee

### ISSUE THREAD

**Issue by austinpilz** [tspringst33n/AncientGates#7]
Hi, I have two servers running BungeeCord #1071 and running AncientGates version 1.9.6

I have followed the instructions to enable Bungee support, which I confirmed by looking in their config files. They did not automatically generate the servers.json file, so I did so myself using the template posted on spigot. Whenever I add a portal from another server, it says "Still connecting to Bungee. Please try again". I have verified that all of my server addresses are correct, the JSON is ok and when I do it, no error is thrown in the console. 

I am using a shared host, but I've never had issues with Bungeecord and Ancient Gates before

**Comment by Gregin** [comment]
I am currently having the same issue, I have 6 servers and only 2 of them work. This is very odd considering some work and some don't and I haven't changed the way I install them. Perhaps the bungee cord server need to have ancient gates?

**Comment by bigyoor** [comment]
hi,

we have the same issues on our bungee servers running on Spigot 1.8.7:

-The plugin don't generate the "servers.json" file. (we added it mannualy but nothing happened)

-when we try to link a portal to an other server the plugin returns in chat:
              "Still connecting to Bungeecord. Try Again" (no console error messages)

-we've tried to downgrade to spigot 1.8.3 and install other versions
 of the plugin but sadly it doesn't work. 

we hope that devloppers will find the problem , this plugin looks really nice!

(sorry for my english if is bad)

**Comment by austinpilz** [comment]
BUMP - I hope the developer could respond to our issues, please. Multiple people have been experiencing the same issue

**Comment by tspringst33n** [comment]
I have been getting reports of that happening. It seems like only 5% of all downloads have this issue. I cannot reproduce it and dont know why it keeps happening.

**Comment by MCTyler** [comment]
Add me to one of the 5 per centers....Just says still connecting to bungeecord.

**Comment by MCTyler** [comment]
Should we delete the password? I don't have a server password.

**Comment by austinpilz** [comment]
I've tried removing / changing the passwords and the error still persists. As for myself, I am on a shared host. Could there be something awry with the ports or firewalls?

**Comment by MCTyler** [comment]
I have a Dedi and it connects VIA LocalHost so it doesn't really go "outside". I downloaded the source and tried to build it with the latest Bukkit and Spigot and its riddled with errors, one of which i could never resolve. Maybe they are in the process of fixing it!

**Comment by MCTyler** [comment]
Who labeled this cant reproduce? Pretty consistent on my end.

**Comment by tspringst33n** [comment]
I was the one who marked it cannot reproduce. I have tried multiple times with multiple setups and cannot reproduce it. I know about 5% of downloads may have this problem and the other 95% don't. I don't know why this is happening. 


On Jul 24, 2015, at 19:03, Steve <notifications@github.com> wrote:

Who labeled this cant reproduce? Pretty consistent on my end.

**Comment by tspringst33n** [comment]
MCTyler, What errors do you get when you build the plugin?

**Comment by MCTyler** [comment]
I will build it again and post them on here. Thanks for the reply.

**Comment by MCTyler** [comment]
Here is the full Stack Trace of the Build....
cd /home/steve/NetBeansProjects/AncientGates; JAVA_HOME=/usr/lib/jvm/java-8-oracle /home/steve/netbeans-8.0/java/maven/bin/mvn install
Scanning for projects...

Some problems were encountered while building the effective model for org.mcteam.ancientgates:AncientGates:jar:1.9.5
'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.spigotmc:spigot-api:jar -> version 1.8.3-R0.1-SNAPSHOT vs 1.8.7-R0.1-SNAPSHOT @ line 66, column 15
'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.bukkit:bukkit:jar -> version 1.8.7-R0.1-SNAPSHOT vs 1.8-R0.1-SNAPSHOT @ line 78, column 15

It is highly recommended to fix these problems because they threaten the stability of your build.

For this reason, future Maven versions might no longer support building such malformed projects.

                                                                        
------------------------------------------------------------------------
Building AncientGates 1.9.5
------------------------------------------------------------------------
Downloading: https://hub.spigotmc.org/nexus/content/groups/public/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.pom

Downloading: http://nexus.theyeticave.net/content/repositories/pub_releases/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.pom

Downloading: https://oss.sonatype.org/content/repositories/snapshots/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.pom

Downloading: http://repo.maven.apache.org/maven2/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.pom

The POM for com.dsh105:EchoPet:jar:2.7.0 is missing, no dependency information available
Downloading: https://hub.spigotmc.org/nexus/content/groups/public/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.jar

Downloading: http://nexus.theyeticave.net/content/repositories/pub_releases/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.jar

Downloading: https://oss.sonatype.org/content/repositories/snapshots/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.jar

Downloading: http://repo.maven.apache.org/maven2/com/dsh105/EchoPet/2.7.0/EchoPet-2.7.0.jar

------------------------------------------------------------------------
BUILD FAILURE
------------------------------------------------------------------------
Total time: 25.989s
Finished at: Sat Jul 25 06:27:54 CDT 2015
Final Memory: 11M/147M
------------------------------------------------------------------------
Failed to execute goal on project AncientGates: Could not resolve dependencies for project org.mcteam.ancientgates:AncientGates:jar:1.9.5: Could not find artifact com.dsh105:EchoPet:jar:2.7.0 in spigot-repo (https://hub.spigotmc.org/nexus/content/groups/public/) -> [Help 1]

To see the full stack trace of the errors, re-run Maven with the -e switch.
Re-run Maven using the -X switch to enable full debug logging.

For more information about the errors and possible solutions, please read the following articles:
[Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

**Comment by MCTyler** [comment]
I redid the project setup and re-factored all the packages and it still cannot resolve this import...import org.mcteam.ancientgates.commands.base.CommandVersion; This causes 2 errors, one in the Plugin.java, and another in CommandHelp.java as follows....
1.)(Plugin.java) import org.mcteam.ancientgates.commands.base.CommandVersion;
2.) (CommandHelp.java) lines.add( new CommandVersion().getUsageTemplate(true, true) );

**Comment by MCTyler** [comment]
And when I try to build again....(of course)
------------------------------------------------------------------------
Building AncientGates 1.9.5
------------------------------------------------------------------------
The POM for com.dsh105:EchoPet:jar:2.7.0 is missing, no dependency information available

--- maven-resources-plugin:2.5:resources (default-resources) @ AncientGates ---
[debug] execute contextualize
Using 'UTF-8' encoding to copy filtered resources.
Copying 1 resource to .

--- maven-compiler-plugin:2.3.2:compile (default-compile) @ AncientGates ---
Compiling 81 source files to /home/steve/NetBeansProjects/AncientGates/target/classes
-------------------------------------------------------------
COMPILATION ERROR : 
-------------------------------------------------------------
org/mcteam/ancientgates/Plugin.java:[55,44] error: cannot find symbol
 package org.mcteam.ancientgates.commands.base
org/mcteam/ancientgates/Plugin.java:[229,19] error: cannot find symbol
 class Plugin
org/mcteam/ancientgates/commands/base/CommandHelp.java:[81,17] error: cannot find symbol
3 errors 
-------------------------------------------------------------
------------------------------------------------------------------------
BUILD FAILURE
------------------------------------------------------------------------
Total time: 14.932s
Finished at: Sat Jul 25 06:55:36 CDT 2015
Final Memory: 19M/219M
------------------------------------------------------------------------
Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project AncientGates: Compilation failure: Compilation failure:
org/mcteam/ancientgates/Plugin.java:[55,44] error: cannot find symbol
package org.mcteam.ancientgates.commands.base
org/mcteam/ancientgates/Plugin.java:[229,19] error: cannot find symbol
class Plugin
org/mcteam/ancientgates/commands/base/CommandHelp.java:[81,17] error: cannot find symbol
-> [Help 1]

**Comment by MCTyler** [comment]
Hopes this leads you to a resolution!

**Comment by Gregin** [comment]
As a solution, use ancient gates for portals within a server and then use bungeeportals for server transportation. It only goes from one server to spawn on the other server and takes awhile to setup, but it is a valid solution in the mean time.

**Comment by MCTyler** [comment]
We just added creative gates back, I have factions on that server already anyway and the bungee portal I used was super easy, just click with axe and type portal add [server]. Thanks!

**Comment by psirusa** [comment]
tspringst33n, did you build this with java 7 or 8, and are you running your test server with 7 or 8?  I can give you access to my Multicraft panel, as I am having the same issue.

**Comment by MCTyler** [comment]
I actually tried both. I just found simplebungee portals and used it and used Creative gates for my other gates. This plugin has way too much dependency requirements.

**Comment by psirusa** [comment]
Yeah, I've run a similar course.  I was offering my setup to help the developer debug :)

**Comment by Gregin** [comment]
I'm just going to accept that this version of Ancient Gates will never be 100% working.
